racksdb/web/ui/src/components/HomeViewCards.vue

```diff
@@ -31,9 +31,9 @@ onMounted(() => {
 </script>
 
 <template>
-  <div class="cards flex justify-around pt-32 px-32">
+  <div class="w-full flex justify-center">
     <div
-      class="w-full max-w-sm p-4 bg-white border border-gray-200 rounded-lg shadow sm:p-8 dark:bg-gray-800 dark:border-gray-700"
+      class="box1 flex flex-col w-[800px] min-h-[500px] bg-white border border-gray-200 rounded-lg shadow sm:p-8 dark:bg-gray-800 dark:border-gray-700 mr-10"
```

Comment:
What is the purpose of `box1` and `box2` classes on the divs? Are they really used?

Reply:
No box1 and box2 aren't used. I used them as references to know where i was in the code but I should have used comments, it would have been more relevant.